src/formDesigner/routes.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from "react";
```

Comment:
This routes.js file was deleted because it is not used. I think you added it back as part of merge. Please delete it.